addons/pos_sale/report/sale_report.py

```diff
@@ -27,6 +27,8 @@ def _select_pos(self):
         select_ = f"""
             -MIN(l.id) AS id,
             l.product_id AS product_id,
+            l.price_unit AS price_unit,
+            l.order_id AS order_id,
```

Comment:
Why you change this? :thinking: 
Sorry for not noticing earlier

Reply:
Yes, for 'price_unit' I want that field in the list view
and I think we have no need to create field of 'order_id' we can use 'order_reference' instead.